web/src/pages/mgmt/modifyBusiness.tsx

```diff
@@ -0,0 +1,208 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { SingleColumnContainer } from "@/components/njwds/SingleColumnContainer";
+import { PageCircularIndicator } from "@/components/PageCircularIndicator";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { LicenseNameTaskIdMapping } from "@businessnjgovnavigator/shared/";
+import {
+  generateLicenseDetails,
+  getCurrentDate,
+  LicenseStatus,
+  licenseStatuses,
+  LicenseTaskID,
+  taskIdLicenseNameMapping,
+  TaskProgress,
+} from "@businessnjgovnavigator/shared/index";
+import { FormControl, InputLabel, MenuItem, Select } from "@mui/material";
+import { useRouter } from "next/router";
+import { ReactElement, useEffect, useState } from "react";
+
+const ModifyBusinessPage = (): ReactElement => {
+  const router = useRouter();
+
+  const { business, updateQueue } = useUserData();
+  const isLicenseDataUndefined = !business?.licenseData;
+
+  const [licenseName, setLicenseName] = useState("");
+  const [licenseStatus, setLicenseStatus] = useState("");
+  const [taskID, setTaskId] = useState("");
+
+  const licenseSearchEnabledTaskIds = Object.keys(taskIdLicenseNameMapping).sort();
+
+  useEffect(() => {
+    if (process.env.NODE_ENV !== "development") {
```

Comment:
[pebble] I'm not super concerned about testing for the functionality in this component, but we should have a test to validate that this doesn't show up on `prod`.